ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartQueryEditor.tsx

```diff
@@ -43,7 +42,11 @@ export function TimeSeriesChartQueryEditor(props: TimeSeriesChartOptionsEditorPr
   const handleQueryChange = (index: number, queryDef: TimeSeriesQueryDefinition) => {
     onChange(
       produce(value, (draft: TimeSeriesChartOptions) => {
-        draft.queries[index] = queryDef;
+        if (draft.queries) {
```

Comment:
need to check if `draft.queries` exists because we don't define the `queries` in the plugin spec anymore